lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -272,10 +274,10 @@ class QcFlagService {
      * Delete single instance of QC flag
      * @param {number} id QC flag id
      * @param {object} [relations] QC Flag entity relations
-     * @param {UserIdentifier+IsAdminOption} [relations.user] user
+     * @param {UserWithRoles} [relations.user] user
```

Comment:
I would rename the relation accordingly, so userWithRoles

Reply:
renamed